frontend/src/app/supportAdmin/AddOrganizationAdmin/AddOrganizationAdminFormContainer.test.tsx

```diff
@@ -84,13 +85,9 @@ const waitForOrgLoadReturnTitle = async () => {
 };
 
 const selectOrg = async () => {
-  // using the default test id that comes with the trusswork component
-  await userEvent.click(screen.getByTestId("combo-box-select"));
-  await userEvent.click(
-    screen.getByTestId(
-      "combo-box-option-DC-Space-Camp-f34183c4-b4c5-449f-98b0-2e02abb7aae0"
-    )
-  );
+  await act(() => {
```

Comment:
⚠️ example usage of `react-select-event` 

[doc on select](https://github.com/romgain/react-select-event/tree/e70973fec5278340b06fc3a6766a464c3de8b063#selectinput-htmlelement-optionoroptions-matcher--arraymatcher-config-object-promisevoid)